Kickstarter-iOS/Views/RewardCardView.swift

```diff
@@ -228,12 +228,16 @@ public final class RewardCardView: UIView {
   }
 
   private func setupConstraints() {
+    let stateImageViewContainerWidthConstraint = self.stateImageViewContainer.widthAnchor
+      .constraint(equalToConstant: Styles.grid(5))
+    stateImageViewContainerWidthConstraint.priority = .defaultHigh
```

Comment:
To be consistent with how we've set priority, could you use this instead?
```
let stateImageViewContainerWidthConstraint = self.stateImageViewContainer.widthAnchor
      .constraint(equalToConstant: Styles.grid(5))
|> \.priority .~ .defaultHigh
```

Reply:
😄 i was gonna merge it and make that comment on your PR!

j/k i'll update it.